indicators/Smma/Smma.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Skender.Stock.Indicators
+{
+    public static partial class Indicator
+    {
+        // SMOOTHED MOVING AVERAGE
+        /// <include file='./info.xml' path='indicators/type[@name="Main"]/*' />
+        ///
+        public static IEnumerable<SmmaResult> GetSmma<TQuote>(
+            IEnumerable<TQuote> history,
+            int lookbackPeriod)
+            where TQuote : IQuote
+        {
+            // sort history
+            List<TQuote> historyList = history.Sort();
+
+            // check parameter arguments
+            ValidateSmma(history, lookbackPeriod);
+
+            // initialize
+            List<SmmaResult> results = new List<SmmaResult>(historyList.Count);
+
+            // roll through history
+            for (int i = 0; i < historyList.Count; i++)
+            {
+                TQuote h = historyList[i];
+                int index = i + 1;
+
+                SmmaResult result = new SmmaResult
+                {
+                    Date = h.Date
+                };
+
+                // first SMMA calculated as simple SMA
+                if (index == lookbackPeriod)
+                {
+                    decimal sumSma = 0m;
```

Comment:
I know I probably use this exact `sumSma` syntax in other indicators, but grammatically, should probably be `sumClose` to be more precise language.  :neckbeard:

Reply:
👍